<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tasky</class>
 <widget class="QMainWindow" name="Tasky">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasky</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Agregar tarea</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="LTarea">
         <property name="text">
          <string>Tarea:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="TxWork">
         <property name="placeholderText">
          <string>Ingrese una tarea. . .</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="LAsig">
         <property name="text">
          <string>Asignatura:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="BoxMat">
         <item>
          <property name="text">
           <string>Calculo Integral</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Electrotecnia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Estructura de Datos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ã‰tica</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Programacion Orientada a Objetos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fisica</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="LDate">
         <property name="text">
          <string>Fecha:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateEdit" name="Eddate">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2023</year>
           <month>12</month>
           <day>18</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>d/M/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="LHora">
         <property name="text">
          <string>Hora:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTimeEdit" name="Edtime"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Tareas</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="TblWorks"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>145</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>318</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="Btnadd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
